src/NexusMods.App.UI/Pages/Sorting/LoadOrder/TreeDataGrid/LoadOrderComponents.cs

```diff
@@ -78,11 +78,28 @@ public static int Compare<TKey>(CompositeItemModel<TKey> a, CompositeItemModel<T
     }
 
     [UsedImplicitly]
-    public sealed class NameColumn : ICompositeColumnDefinition<NameColumn>
+    public sealed class DisplayNameColumn : ICompositeColumnDefinition<DisplayNameColumn>
     {
-        public const string ColumnTemplateResourceKey = nameof(LoadOrderColumns) + "_" + nameof(NameColumn);
+        public const string ColumnTemplateResourceKey = nameof(LoadOrderColumns) + "_" + nameof(DisplayNameColumn);
+
+        public static readonly ComponentKey DisplayNameComponentKey = ComponentKey.From(ColumnTemplateResourceKey + "_" + "DisplayNameComponent");
+
+        public static int Compare<TKey>(CompositeItemModel<TKey> a, CompositeItemModel<TKey> b) where TKey : notnull
+        {
+            throw new NotImplementedException();
+        }
+
+        // The header name should be set on column creation as it is game dependent
+        public static string GetColumnHeader() => throw new NotSupportedException();
```

Comment:
We should set the column header here directly for ModNameColumn instead of passing it in later from the sortableItemProviderFactory, since it shouldn't be game dependent.